compio-driver/src/iour/mod.rs

```diff
@@ -49,36 +61,21 @@ pub trait OpCode {
     }
 }
 
-/// Low-level driver of io-uring.
-pub(crate) struct Driver {
-    inner: IoUring,
-    squeue: VecDeque<squeue::Entry>,
-    notifier: Notifier,
-    notifier_registered: bool,
-    pool: AsyncifyPool,
-    pool_completed: Arc<SegQueue<Entry>>,
+struct UnlimitedUring<S: squeue::EntryMarker, C: cqueue::EntryMarker> {
```

Comment:
What does unlimit mean? Unbounded as in vec deque size?

Reply:
The submission queue will be unbounded. I'll change its name to `UnboundedUring`.